src/app/lib/utilities/srcSet/index.test.js

```diff
@@ -4,54 +4,54 @@ describe('create srcset', () => {
   const srcsetScenarios = [
     {
       originCode: 'cpsdevpb',
-      locator: 'testland',
+      locator: 'testland.jpg',
       width: 1024,
       expected: {
         primarySrcset:
-          'https://ichef.bbci.co.uk/ace/ws/240/cpsdevpb/testland.webp 240w, https://ichef.bbci.co.uk/ace/ws/320/cpsdevpb/testland.webp 320w, https://ichef.bbci.co.uk/ace/ws/480/cpsdevpb/testland.webp 480w, https://ichef.bbci.co.uk/ace/ws/624/cpsdevpb/testland.webp 624w, https://ichef.bbci.co.uk/ace/ws/800/cpsdevpb/testland.webp 800w',
+          'https://ichef.bbci.co.uk/ace/ws/240/cpsdevpb/testland.jpg.webp 240w, https://ichef.bbci.co.uk/ace/ws/320/cpsdevpb/testland.jpg.webp 320w, https://ichef.bbci.co.uk/ace/ws/480/cpsdevpb/testland.jpg.webp 480w, https://ichef.bbci.co.uk/ace/ws/624/cpsdevpb/testland.jpg.webp 624w, https://ichef.bbci.co.uk/ace/ws/800/cpsdevpb/testland.jpg.webp 800w',
         primaryMimeType: 'image/webp',
         fallbackSrcset:
-          'https://ichef.bbci.co.uk/ace/ws/240/cpsdevpb/testland 240w, https://ichef.bbci.co.uk/ace/ws/320/cpsdevpb/testland 320w, https://ichef.bbci.co.uk/ace/ws/480/cpsdevpb/testland 480w, https://ichef.bbci.co.uk/ace/ws/624/cpsdevpb/testland 624w, https://ichef.bbci.co.uk/ace/ws/800/cpsdevpb/testland 800w',
-        fallbackMimeType: null,
+          'https://ichef.bbci.co.uk/ace/ws/240/cpsdevpb/testland.jpg 240w, https://ichef.bbci.co.uk/ace/ws/320/cpsdevpb/testland.jpg 320w, https://ichef.bbci.co.uk/ace/ws/480/cpsdevpb/testland.jpg 480w, https://ichef.bbci.co.uk/ace/ws/624/cpsdevpb/testland.jpg 624w, https://ichef.bbci.co.uk/ace/ws/800/cpsdevpb/testland.jpg 800w',
+        fallbackMimeType: 'image/jpeg',
       },
       summary:
-        'should return a srcset with test in originCode and testland in location',
+        'should return a srcset with test in originCode and testland in location', // why was fallbackMimeType set to null in this test?
```

Comment:
```suggestion
        'should return a srcset with test in originCode and testland in location',
```
We haven't resolved this mystery, lets drop the comment?

Reply:
done